include/crm/common/results.h

```diff
@@ -294,6 +287,13 @@ typedef enum crm_exit_e {
     CRM_EX_DEGRADED             = 190, //!< Service active but more likely to fail soon
     CRM_EX_DEGRADED_PROMOTED    = 191, //!< Service promoted but more likely to fail soon
 
+    /* Custom
+     *
+     * This can be used to initialize exit status variables or to indicate that
+     * a command is pending (which is what the controller uses it for).
+     */
+    CRM_EX_NONE                 = 193, //!< No exit status available
```

Comment:
Do you want to save some more space between CRM_EX_DEGRADED_PROMOTED and CRM_EX_NONE for future use?

Reply:
This is a replacement for PCMK_OCF_UNKNOWN, so it's already a well-established code